src/toolbar/toolbar-title.jsx

```diff
@@ -14,6 +14,10 @@ function getStyles(props, state) {
       fontSize: toolbar.titleFontSize,
       display: 'inline-block',
       position: 'relative',
+      textOverflow: 'ellipsis',
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      width: (props.text.length > 40) ? 150 : 'inherit',
```

Comment:
I'm asking this because the `AppBar` component is not provinding any width. `40` and `150` look like magic numbers.
What if we where using `flexbox` here?